sdks/python/apache_beam/transforms/display_test.py

```diff
@@ -78,6 +89,16 @@ def matches_kvn(cls, key, value, namespace):
                namespace=namespace,
                match_props=['key', 'value', 'namespace'])
 
+  @classmethod
+  def matches_all(cls, key, value, namespace, label, shortValue):
```

Comment:
Using match_props and adding a new method for every subset doesn't seem very extensible. What if instead you just had

```
class DisplayDataItemMatcher(...):
  IGNORED = object()
  def __init__(self, key, value=IGNORED, namespace=IGNORED, ...)
    ...

```

and so DisplayDataItemMatcher(...) simply matched the keywords passed.